### PR TITLE
Add config file name explanation in README (Close #264)

### LINKED ISSUE #264: Add an explanation about the config file name inside the Readme
## What's wrong

By reading the Readme from top to bottom, we are given [an example of config file](https://github.com/meilisearch/docs-scraper/issues/263) but it is not clear in which file we have to put this.

The information can be found below in an other section (the config file can be anything as it is passed as an argument later), but the information is arriving later than what we would expect.

## Solution

Explain [inside the config file paragraph](https://github.com/meilisearch/docs-scraper/issues/263) that the config file name can be anything and doesn't matter.

### PR DESCRIPTION
**Pull request by 3t8**
# Pull Request

## Related issue
Fixes #264 

## What does this PR do?
- Explain the config file name can be anything as it will be passes as an argument

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ _Meilisearch is open-source and can run either on your server or on any cloud pr
 
 ### Set your Config File
 
-The scraper tool needs a config file to know which content you want to scrape. This is done by providing **selectors** (e.g. the HTML tag/id/class).
+The scraper tool needs a config file to know which content you want to scrape. This is done by providing **selectors** (e.g. the HTML tag/id/class). The config file will be passed as an argument, so you can use any name you like.
```
Comment (alallema) [inline README.md:90]:
```suggestion
The scraper tool needs a config file to know which content you want to scrape. This is done by providing **selectors** (e.g. the HTML tag/id/class). The config file will be passed as an argument, so you can name your file whatever you want.
```

**Review by alallema — COMMENTED** [review]
Thanks, @3t8 for this PR!
I made a suggestion on your proposition

**Comment by 3t8** [comment]
@alallema updated with your suggestion

File: README.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ _Meilisearch is open-source and can run either on your server or on any cloud pr
 
 ### Set your Config File
 
-The scraper tool needs a config file to know which content you want to scrape. This is done by providing **selectors** (e.g. the HTML tag/id/class).
+The scraper tool needs a config file to know which content you want to scrape. This is done by providing **selectors** (e.g. the HTML tag/id/class). The config file will be passed as an argument, so you can name your file whatever you want.
```
Comment (alallema) [inline README.md:90]:
```suggestion
The scraper tool needs a config file to know which content you want to scrape. This is done by providing **selectors** (e.g. the HTML tag/id/class). The config file IS passed as an argument. It follows no naming convention and may be named as you want.
```

**Review by alallema — COMMENTED** [review]
Hi @3t8,
So sorry I realize the sentence I suggested wasn't so great so there is a new one 😊

**Review by alallema — APPROVED** [review]
LGTM! 🎉

**Comment by alallema** [comment]
bors merge

**Pull request merged**